src/components/variable.jsx

```diff
@@ -232,15 +241,83 @@ const Variable = React.createClass({
       return substring
     })
   },
-  // Change every OpenFisca variable in the formula by a link to the variable page:
-  renderLinkedFormulaVariables(formula) {
-    // Split on double quotes first (preventing collision with Link):
-    const splits = this.splitAndLink(formula, '"')
-    return splits.map((substring) => {
-      // Only split strings, as trying to split JSX Links would raise an error
-      return typeof substring == 'string' ? this.splitAndLink(substring, '\'') : substring
+
+  splitAndLinkParams(text) {
```

Comment:
Export formulas operations in a new js file to avoid referring to parameters in a file named `variables.jsx`? This new file would manage links on OpenFisca parameters and variables.

Reply:
Yep, extracting this complex logic in a `formula.js` file or in a `formula.jsx` component could be nice.

Reply:
Done 😫